test/plugins/fingerprint.plugin.spec.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
This gets to a more general trend in our tests: We have to provide parameters to our mock methods in order to satisfy the interface requirements, but we never actually use them.

I'd argue that we should change this at some point, but it's out of scope for this PR. Will remove this disabling, though, so that we can keep an eye on this.